mappings/net/minecraft/datafixer/fix/ChunkPalettedStorageFix.mapping

```diff
@@ -41,6 +48,10 @@ CLASS net/minecraft/class_3582 net/minecraft/datafixer/fix/ChunkPalettedStorageF
 		METHOD method_15640 get (III)I
 			ARG 1 x
 			ARG 2 y
+		METHOD method_15641 isLowBits (I)Z
```

Comment:
like `index & (-index) == index`?

Reply:
No, it's whether the value at the index is stored in the low bits of the value in the packed integer array: `(index & 0x1) == 0x0`

Reply:
What you wrote is like `isEven` instead, or do you mean `(array[index] & 1) == 0`? Afaik low bit means something like `Integer.lowestOneBit()`

Reply:
By "low bits" I mean that each byte in the backing byte array is split into 2 groups of 4 bits, the low bits and the high bits (perhaps "low nibble" would be clearer?). Just because the method returns true when the input is even doesn't mean that it should be called `isEven`, it's used along with the other method `getRawIndex`, which equally shouldn't be called `divideByTwo`. `getRawIndex` gets the index into the backing byte array your index refers to, and this method (currently called `isLowBits`) returns true when your index refers to the low nibble of the byte in that byte array, and false if it's the high nibble.

Reply:
I'd suggest calling it "low nibble", or just call this method `isLower` (you already have the param named index so it should be clear)

Reply:
:+1: renamed it to `usesLowNibble`